### DISCUSSION TITLE
Sequence diagram: Parallel(Async) Subsequences

### DISCUSSION THREAD

**Discussion by bmsan** [mermaid-js/mermaid#3903] [Q&A]
Hello, 
I would like to be able to draw two "subsequences" which are running in parallel (unsynchronized between them).
The end result that I would want to achive can be seen in the below image, where the two blocks (yellow and purple) are drawn in parallel.
The timing of events inside the yellow block is independent of the timing of events inside the purple block.
![image](https://user-images.githubusercontent.com/52523668/206910993-c414099e-eb19-4809-b8c8-b41f04197b4f.png)

Would this be possible with mermaid?

**Comment by Mauroquil-bit** [comment]
Te puede llegar a servir esto:

`::: mermaid
    graph TD
    
    A[Secuencia 1] --> B[Secuencia 2]
    B --> C
    C --> D
    D --> E

    F[Secuencia 3] --> G[Secuencia 4]
    G --> H
    H --> I
    I --> J
:::
`
Luego te conviene seguir por este tutorial:   https://mermaid.js.org/syntax/sequenceDiagram.html
Por último, en este link podrás hacer tus borradores:  https://mermaid.live/edit
